35.py

```diff
@@ -0,0 +1,25 @@
+# n = len(nums)
+# time:O(n)
+# space:O(1)
+
+class Solution(object):
+    def searchInsert(self, nums: List[int], target: int) -> int:
+        """A function to  return the index if the target is found. If not, return the index where it would be if it were inserted in order.
+        Args:
+            nums (List[int]): A given a sorted array of distinct integers.
+            target (int): A given an integer.
+        Returns:
+            int: The index if the target is found. If not, return the index where it would be if it were inserted in order.
+        """
+        r=0
+        l=len(nums)-1
```

Comment:
The variable name "l" is inappropriate
https://www.flake8rules.com/rules/E741.html#:~:text=Variables%20named%20I%20%2C%20O%20%2C%20and,variables%20to%20something%20more%20descriptive.